src/main/java/pro/cloudnode/smp/bankaccounts/Account.java

```diff
@@ -132,9 +132,7 @@ public final void updateBalance(final @NotNull BigDecimal diff) {
     public final @NotNull Transaction transfer(final @NotNull Account to, final @NotNull BigDecimal amount, final @Nullable String description, final @Nullable String instrument) {
         if (frozen) throw new IllegalStateException("Your account is frozen");
         if (to.frozen) throw new IllegalStateException("Recipient account is frozen");
-        if (amount.compareTo(BigDecimal.ZERO) <= 0) throw new IllegalArgumentException("Amount must be greater than zero");
```

Comment:
i think we should keep this here

Reply:
well the problem with that is when a plugin does a negative transfer (to remove money), this check would fail

Reply:
This should be in the vault Economy class imo. Negative transfers within our system should not be allowed